tests/virt/upgrade/conftest.py

```
@@ -275,32 +289,33 @@ def run_strategy_golden_image_data_source(admin_client, run_strategy_golden_imag
+            for condition in vm.vmi.instance.status.conditions
+        ):
+            migratable_vms.append(vm)
 
-    LOGGER.info(f"VIRT migratable vms: {[vm.name for vm in migratable_vms]}")
     return migratable_vms
 
 
+@pytest.fixture(scope="session")
+def virt_migratable_vms_names(virt_migratable_vms):
+    vm_names = [vm.name for vm in virt_migratable_vms]
+    LOGGER.info(f"All migratable vms: {vm_names}")
```

Ruff (0.14.11)

Logging statement uses f-string

(G004)